docs/backup_and_restore.rst

```diff
@@ -123,67 +105,216 @@ Restoring
 Prerequisites
 '''''''''''''
 
-The process for restoring a backup is very similar to the process of creating
-one. As before, boot the *Admin Workstation* and ``cd`` to the
-SecureDrop repository. Ensure that you have SecureDrop 0.4 or later
-checked out.
+To perform a restore, boot into the *Admin Workstation* and
+ensure that your ``.tar.gz`` backup archive has been copied to
+``~/Persistent/securedrop/install_files/ansible-base``.
+(If you are using the same *Admin Workstation* as you did when you took the backup,
+the archive will already be in place).
 
-The restore command expects to find a ``.tar.gz`` backup archive in
-``install_files/ansible-base`` under the SecureDrop repository root directory.
-If you are using the same *Admin Workstation* to do a restore from a previous
-backup, it should already be there because it was placed there by the backup
-command. Otherwise, you should copy the backup archive that you wish to restore to
-``install_files/ansible-base``.
+If you are restoring data onto an existing instance whose onion URLs have not
+changed (for example, for data recovery purposes), see :ref:`Restoring Data on an
+Existing SecureDrop Installation <restore_data>`.
+
+If you are restoring data onto an instance whose onion URLs *have* changed,
+or you are performing a hardware migration, see :ref:`Migrating <migrating>`. 
```

Comment:
The phrase "onion URLs have changed" seems ambiguous to me (relative to what?), and I think it could be difficult for an admin to choose between the "Migrating" pointer and the following "mismatched Tor config" pointer based on how this is currently worded. As I understand it, we want to cover two scenarios in this chapter:

- restoring data for an existing, working installation
- reinstalling and then restoring from a backup, either for hardware/platform migration or for disaster recovery

Depending on when the last backup was taken, both scenarios may require the use of `--preserve-tor-config` to avoid clobbering a working Tor config on the server. If that's accurate, I would suggest structuring the chapter into these two sections at a high level, and explaining the `--preserve-tor-config` behavior in context where appropriate.

Reply:
Thank you for this feedback! In 151175d I've described the instructions as you indicate, and I've moved the section on `--preserve Tor config` to a section of "Additional Instructions" at the end. (Eventually I will include a small section in that Additionals section on rebuilding the monitor server, as per e.g. https://github.com/freedomofpress/securedrop/issues/5311, but for now the only thing in there is the `preserve Tor config` piece.)